pkg/api/bzz.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"hash"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	bmtlegacy "github.com/ethersphere/bmt/legacy"
+	"github.com/gorilla/mux"
+	"golang.org/x/crypto/sha3"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+type bzzPostResponse struct {
+	Hash string
```

Comment:
I would suggest to have this field as 
```
Address swarm.Address `json:"address"`
```
 It will automatically be marshaled and we will preserve the type information for clarity on the response. Also to add json tag and to mane it address as this is the name that we used so far for swarm addresses.

Reply:
I originally named this Address but later wondered if this should be so... due to the fact we still are introducing the notion of content addressed data, should reference to actual data be called `Hashes` or also `Addresses`? although they reuse the same underlying data structure to signify where the data should reside, there are clear distinctions in the semantics of content addresses and peer addresses, and so the question is if we should distinguish the two in the codebase. This could potentially have better clarity about whether some code handles content or peer addresses (although this should also be clear from the code itself... nevertheless...).
So this was just a thought experiment, albeit an unsuccessful one :) I'll change it to `Address`

Reply:
Yes, I see your point. It does make sense to name it hash and stick with that name in exposed APIs for chunk/file addressing. Another suggestion from me would be something like this:

```
Hash swarm.Address `json:"hash"`
```

WDYT? Maybe this is more clear that the address is related to chunk, not peer.

Reply:
i addressed your change @janos. we can change later to `Address` if this is needed